mps/mps_test.py

```diff
@@ -0,0 +1,203 @@
+# Copyright 2019 The TensorNetwork Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+import tensornetwork
+import pytest
+import numpy as np
+
+import tensorflow as tf
+
+from jax.config import config
+
+config.update("jax_enable_x64", True)
+tf.compat.v1.enable_v2_behavior()
+
+
+def get_random_np(shape, dtype):
```

Comment:
For all the random operations, I suggest to add a fixed seed to ensure reproducibility of tests

Reply:
We've been burned by this before.

Reply:
I've added the seed to the initializer